### ISSUE TITLE
[BUG] Error in case of CLASS DEFINITION DEFERRED

### ISSUE THREAD

**Issue by mraht** [SAP/abap-cleaner#294]
If source code contains the following statement with CLASS ... DEFINITION DEFERRED like the following code snippet, an error occurs and execution of abap cleaner is being cancelled:
```
CLASS: 
  lcl_exits DEFINITION DEFERRED,
  lcl_application DEFINITION DEFERRED,
  lcl_event_receiver DEFINITION DEFERRED.
```

![image](https://github.com/SAP/abap-cleaner/assets/99023738/82435600-b4f9-4b2c-9cfb-217f6cf63ad1)

**Comment by jmgrassau** [comment]
Hi Marius, 

thanks for reporting this – yes, this is actually an unnecessary limitation and will be fixed with the next release. Until then, you could simply change this "CLASS: ..." chain into three "CLASS ... DEFINITION DEFERRED." statements, as suggested by the error message. 

But regardless of this, you can always do nasty things with chains such as:

```ABAP
REPORT any_report.

START-OF-SELECTION.
  WRITE 'Hello world'.
  CLASS: lcl_any DEFINITION DEFERRED,
         lcl_other DEFINITION DEFERRED,
  lcl_any DEFINITION.

PUBLIC SECTION.
ENDCLASS.

CLASS lcl_other DEFINITION.
  PUBLIC SECTION.
ENDCLASS.
```

The above example was pretty-printed, but you can see how even the Pretty Printer is struggling to make sense of this. ABAP cleaner won't process such a case, either, although it is syntactically correct. 

Interestingly, CLASS ... DEFINITION DEFERRED doesn't end the START-OF-SELECTION event, so if you run this:

```ABAP
REPORT any_report.

START-OF-SELECTION.
  WRITE / 'Hello world'.
  CLASS: lcl_any DEFINITION DEFERRED,
         lcl_other DEFINITION DEFERRED.
  WRITE / 'Hello again'.

CLASS lcl_any DEFINITION.
  PUBLIC SECTION.
ENDCLASS.

CLASS lcl_other DEFINITION.
  PUBLIC SECTION.
ENDCLASS.
```

you will get:

```
Hello world
Hello again
```

meaning that CLASS: ... is correctly indented with 2 spaces there. 

Kind regards,
Jörg-Michael

**Comment by jmgrassau** [comment]
Hi Marius, 

thanks again for reporting this bug – this should now be fixed with version 1.16.1, which was just released!

Kind regards,
Jörg-Michael
